### PR TITLE
renamed functions that log and fetch pickle objects from artifacts to make it consistent with other log_{} functions

### PR DESCRIPTION
**Pull request by jakubczakon**
Changes:
* renamed `pickle_and_log_artifact` to `log_pickle`
* renamed `get_pickled_artifact` to `get_pickle`

### CODE REVIEW

File: neptunecontrib/api/utils.py [hunk 1]
```diff
@@ -135,7 +137,8 @@ def extract_project_progress_info(leadearboard, metric_colname, time_colname='fi
     progress_df = _get_daily_experiment_counts(progress_df)
     progress_df = _get_current_best(progress_df)
     progress_df = progress_df[
-        ['id', 'metric', 'metric_best', 'running_time', 'running_time_day', 'experiment_count_day',
+        ['id', 'metric', 'metric_best', 'running_time', 'running_time_day',
```
Comment (pitercl) [inline neptunecontrib/api/utils.py:140]: Why are we breking this and other similar lines?

File: neptunecontrib/api/utils.py [hunk 2]
```diff
@@ -272,47 +279,58 @@ def pickle_and_log_artifact(obj, filename, experiment=None):
...
 
     Args:
-        experiment(`neptune.experiments.Experiment`): Neptune experiment.
-        filename(str): filename under which object was saved in Neptune.
+        filename(str): filename under which object will be saved to Neptune.
+        experiment (:obj:`neptune.experiments.Experiment`, optional, default is ``None``):
+            | For advanced users only. Pass Neptune
+              `Experiment <https://docs.neptune.ai/neptune-client/docs/experiment.html#neptune.experiments.Experiment>`_
+              object if you want to control to which experiment data is logged.
+            | If ``None``, log to currently active, and most recent experiment.
```
Comment (pitercl) [inline neptunecontrib/api/utils.py:316]: This description seems to be copied from log_ and doesn't quite fit :)

File: neptunecontrib/api/__init__.py [hunk 3]
```diff
@@ -25,23 +25,23 @@
     get_property_columns,
     get_system_columns,
     strip_prefices,
-    pickle_and_log_artifact,
```
Comment (pitercl) [inline neptunecontrib/api/__init__.py:28]: If we remove it here, then the deprecation notices are a bit difficult to access :)

Comment (jakubczakon) [reply neptunecontrib/api/__init__.py:28]: fixed

File: neptunecontrib/api/utils.py (hunk 2 above)
Comment (jakubczakon) [reply neptunecontrib/api/utils.py]: changed to the old one -> I think this should probably be standardized across the lib

File: neptunecontrib/api/utils.py (hunk 1 above)
Comment (jakubczakon) [reply neptunecontrib/api/utils.py]: some autoformat, fixed.

**Pull request merged**